docs/articles/testing/multi-node-testing.md

```diff
@@ -62,13 +67,13 @@ For example: if you're taking advantage of the `akka.cluster.roles` property to
 
 The `NodeConfig` method allows you to do just that:
 
-```csharp
-NodeConfig(new List<RoleName> { First }, 
-    new List<Config> {     ConfigurationFactory.ParseString(
-        @"akka.cluster.roles =[""a"", ""c""]") });
-NodeConfig(new List<RoleName> { Second, Third }, 
-    new List<Config> { ConfigurationFactory.ParseString(
-        @"akka.cluster.roles =[""b"", ""c""]") });
+```c#
```

Comment:
For a really small theoretical code snippet like this, it's probably fine to just let this stand alone as a markdown block rather than a DocFx-style block.